libraries/RC_Channel/RC_Channel.h

```diff
@@ -159,7 +161,9 @@ class RC_Channels {
     static bool set_overrides(int16_t *overrides, const uint8_t len);  // set multiple overrides at once
 
     static void set_pwm_all(void);
-    
+
+    static void set_pwn_slew_rate_limit(uint16_t mask, float slew_rate, float dt);
```

Comment:
I think a better (and more acceptable upstream) api here would be

`limit_slew_rate(uint16_t ch, float slew_rate, float dt);`

And to use this functionality in vehicle code:

```
set_pwm_all();
for (ch in masked channels) {
    limit_slew_rate(ch)
}
```

Note that this will work because the channels are not actually written to the HAL in set_pwm_all().

Reply:
```
set_pwm_all() {
for ch in channels:
   ch.last_pwm = ch.current_pwm;
    ch.current_pwm = new_pwm;
}
```

```
limit_slew_rate() {
 compare last_pwm to current_pwm and adjust current_pwm accordingly
}
```

Reply:
This will also allow setting different slew rates on different channels